misc/config_tools/data/tgl-folder-spc-7100-i7/tgl-vecow-spc-7100-i7.xml

```diff
@@ -0,0 +1,5181 @@
+	  4017000000-4017000fff : 0000:00:10.0
+	    4017000000-40170001ff : lpss_dev
+	      4017000000-40170001ff : i2c_designware.0 lpss_dev
+	    4017000200-40170002ff : lpss_priv
+	  4017001000-4017001fff : 0000:00:15.0
+	    4017001000-40170011ff : lpss_dev
+	      4017001000-40170011ff : i2c_designware.1 lpss_dev
+	    4017001200-40170012ff : lpss_priv
+	    4017001800-4017001fff : idma64.1
+	      4017001800-4017001fff : idma64.1 idma64.1
+	  4017002000-4017002fff : 0000:00:15.1
+	    4017002000-40170021ff : lpss_dev
+	      4017002000-40170021ff : i2c_designware.2 lpss_dev
+	    4017002200-40170022ff : lpss_priv
+	    4017002800-4017002fff : idma64.2
+	      4017002800-4017002fff : idma64.2 idma64.2
+	  4017003000-4017003fff : 0000:00:15.2
+	    4017003000-40170031ff : lpss_dev
+	      4017003000-40170031ff : i2c_designware.3 lpss_dev
+	    4017003200-40170032ff : lpss_priv
+	    4017003800-4017003fff : idma64.3
+	      4017003800-4017003fff : idma64.3 idma64.3
+	  4017004000-4017004fff : 0000:00:15.3
+	    4017004000-40170041ff : lpss_dev
+	      4017004000-40170041ff : i2c_designware.4 lpss_dev
+	    4017004200-40170042ff : lpss_priv
+	    4017004800-4017004fff : idma64.4
+	      4017004800-4017004fff : idma64.4 idma64.4
+	  4017005000-4017005fff : 0000:00:19.0
+	    4017005000-40170051ff : lpss_dev
+	      4017005000-40170051ff : i2c_designware.5 lpss_dev
+	    4017005200-40170052ff : lpss_priv
+	    4017005800-4017005fff : idma64.5
+	      4017005800-4017005fff : idma64.5 idma64.5
+	  4017006000-4017006fff : 0000:00:19.1
+	    4017006000-40170061ff : lpss_dev
+	      4017006000-40170061ff : i2c_designware.6 lpss_dev
+	    4017006200-40170062ff : lpss_priv
+	    4017006800-4017006fff : idma64.6
+	      4017006800-4017006fff : idma64.6 idma64.6
+	  4017007000-4017007fff : 0000:00:1e.0
+	    4017007000-40170071ff : lpss_dev
+	      4017007000-401700701f : serial
+	    4017007200-40170072ff : lpss_priv
+	    4017007800-4017007fff : idma64.7
+	      4017007800-4017007fff : idma64.7 idma64.7
+	  4017008000-4017008fff : 0000:00:1e.3
+	    4017008000-40170081ff : lpss_dev
+	      4017008000-40170081ff : pxa2xx-spi.8 lpss_dev
+	    4017008200-40170082ff : lpss_priv
+	    4017008800-4017008fff : idma64.8
+	      4017008800-4017008fff : idma64.8 idma64.8
+	  4020000000-40ffffffff : 0000:00:02.0
+	  6000000000-6000ffffff : 0000:00:02.0
+	  6001000000-60010fffff : 0000:00:1f.3
+	    6001000000-60010fffff : ICH HD audio
+	  6001100000-600110ffff : 0000:00:14.0
+	    6001100000-600110ffff : xhci-hcd
+	  6001110000-600111ffff : 0000:00:0d.0
+	    6001110000-600111ffff : xhci-hcd
+	  6001120000-6001123fff : 0000:00:1f.3
+	    6001120000-6001123fff : ICH HD audio
+	  6001124000-6001127fff : 0000:00:14.2
+	  6001128000-60011280ff : 0000:00:1f.4
+	  600112d000-600112dfff : 0000:00:16.0
+	    600112d000-600112dfff : mei_me
+	  6001132000-6001132fff : 0000:00:14.2
+	</IOMEM_INFO>
+  <BLOCK_DEVICE_INFO>
+	/dev/sda2: TYPE="ext4"
+	/dev/sda3: TYPE="ext4"
```

Comment:
This is odd, how did you set-up the system before you ran the `board_inspector` tool?

I have only one ext4 partition on my Ubuntu 20.04 installation. The other difference is my storage is on an NVMe drive, not SATA. What is the exact model of the Vecow TGL system this was captured on?